src/Test/L0/Worker/SetEnvFileCommandL0.cs

```diff
@@ -183,6 +183,47 @@ public void SetEnvFileCommand_Simple_SpecialCharacters()
             }
         }
 
+
+        [Fact]
+        [Trait("Level", "L0")]
+        [Trait("Category", "Worker")]
+        public void SetOutputFileCommand_BlockListItemsFiltered()
```

Comment:
do you need to update the test to verify the ENV instead of output?

Reply:
Fixed the test name. The test body used `_setEnvFileCommand.ProcessCommand`